From f2d8bfca6788f68ce878027184f4c7325a5c75d6 Mon Sep 17 00:00:00 2001
From: zachmann <gabriel.zachmann@kit.edu>
Date: Wed, 5 Oct 2022 13:02:48 +0200
Subject: [PATCH] fix macos build

---
 src/ipc/ipc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ipc/ipc.c b/src/ipc/ipc.c
index f92dd284..901b801f 100644
--- a/src/ipc/ipc.c
+++ b/src/ipc/ipc.c
@@ -422,7 +422,7 @@ oidc_error_t ipc_vwrite(SOCKET _sock, const char* fmt, va_list args) {
 #ifdef MINGW
   int written_bytes = send(_sock, msg, msg_len, 0);
 #else
-#ifndef ANY_MSYS
+#ifdef __linux__
   if (fcntl(_sock, F_GETPIPE_SZ) < (int)msg_len) {
     fcntl(_sock, F_SETPIPE_SZ, msg_len);
   }
