From 8a7b0fef856b559053c827888e587b460e9fd88e Mon Sep 17 00:00:00 2001
From: Chris Burr <christopher.burr@cern.ch>
Date: Fri, 18 Nov 2022 10:50:53 +0100
Subject: [PATCH] Use pkg-config to find more libraries

---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index a0e4f29c..701f6ad3 100644
--- a/Makefile
+++ b/Makefile
@@ -96,15 +96,15 @@ else
 endif
 
 
-LSODIUM = -lsodium
+LSODIUM = $(shell pkg-config --libs libsodium)
 LARGP   = -largp
-LMICROHTTPD = -lmicrohttpd
+LMICROHTTPD = $(shell pkg-config --libs libmicrohttpd)
 LCURL = -lcurl
 LSECRET = -lsecret-1
 LGLIB = -lglib-2.0
 LLIST = -llist
 LCJSON = -lcjson
-LQR = -lqrencode
+LQR = $(shell pkg-config --libs libqrencode)
 LAGENT = -l:$(SHARED_LIB_NAME_FULL)
 ifdef ANY_MSYS
 	ifdef MINGW32
-- 
2.38.1

