From 61d51593d26b897b234185984819f21e4ae6171a Mon Sep 17 00:00:00 2001
From: Chris Burr <christopher.burr@cern.ch>
Date: Fri, 18 Nov 2022 11:38:11 +0100
Subject: [PATCH] Use LDFLAGS for linking on macOS

---
 Makefile | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index 701f6ad3..a032f4ce 100644
--- a/Makefile
+++ b/Makefile
@@ -152,15 +152,16 @@ TEST_CFLAGS = $(CFLAGS) -I.
 # Linker options
 LINKER   := $(CC)
 LINKER_XX   := $(CXX)
+LFLAGS := $(LDFLAGS)
 ifdef MAC_OS
-LFLAGS   = $(LSODIUM) $(LARGP)
+LFLAGS   += $(LSODIUM) $(LARGP)
 PROMPT_LFLAGS += $(LFLAGS) -framework WebKit
 else
 ifdef ANY_MSYS
-LFLAGS   = $(LMINGW) $(LSODIUM) $(LARGP)
+LFLAGS   += $(LMINGW) $(LSODIUM) $(LARGP)
 PROMPT_LFLAGS = $(LFLAGS)
 else
-LFLAGS   := $(LDFLAGS) $(LSODIUM) -fno-common
+LFLAGS   += $(LSODIUM) -fno-common
 PROMPT_LFLAGS = -fPIE $(LFLAGS) $(CPPFLAGS)
 ifeq ($(USE_ARGP_SO),1)
 	LFLAGS += $(LARGP)
@@ -187,10 +188,10 @@ endif
 GEN_LFLAGS = $(LFLAGS) $(LMICROHTTPD) $(LQR)
 ADD_LFLAGS = $(LFLAGS)
 ifdef MAC_OS
-CLIENT_LFLAGS = -L$(APILIB) $(LARGP) $(LAGENT) $(LSODIUM)
+CLIENT_LFLAGS = $(LFLAGS) -L$(APILIB) $(LARGP) $(LAGENT) $(LSODIUM)
 else
 ifdef MSYS
-CLIENT_LFLAGS = $(LMINGW) -L$(APILIB) $(LARGP) $(LAGENT) $(LSODIUM)
+CLIENT_LFLAGS =  $(LFLAGS) $(LMINGW) -L$(APILIB) $(LARGP) $(LAGENT) $(LSODIUM)
 else
 CLIENT_LFLAGS := $(LDFLAGS) -L$(APILIB) $(LAGENT) $(LSODIUM)
 ifeq ($(USE_ARGP_SO),1)
-- 
2.37.1 (Apple Git-137.1)

